_themes/conan/static/css/theme.css

```diff
@@ -18,4 +18,15 @@
     font-size:0.8em
  }
 
+ .container.examples ul{
+    margin-left: 10px;
+    list-style:none;
+ }
+
+ .container.examples li:before{
+    content: '\1F4D8';
+    margin-left: -20px;
+    margin-right: 10px;
+ }
+
```

Comment:
I missed to do some "art"
![image](https://user-images.githubusercontent.com/3807515/167168126-1c510516-6b9c-4f4b-8771-f3ea75c78487.png)